Microsoft.Toolkit.Uwp.UI.Behaviors/InfoBar/StackedNotificationsBehavior.cs

```diff
@@ -42,10 +42,39 @@ public StackedNotificationsBehavior()
         /// <param name="notification">The notification to display.</param>
         public void Show(Notification notification)
         {
-            _stackedNotifications.Enqueue(notification);
+            if (notification is null)
+            {
+                throw new ArgumentNullException(nameof(notification));
+            }
+
+            _stackedNotifications.AddLast(notification);
             ShowNext();
         }
 
+        /// <summary>
+        /// Remove the <paramref name="notification"/>.
+        /// If the notification is displayed, it will be closed.
+        /// If the notification is still in the queue, it will be removed.
+        /// </summary>
+        /// <param name="notification">The notification to remove.</param>
+        public void Remove(Notification notification)
+        {
+            if (notification is null)
+            {
+                throw new ArgumentNullException(nameof(notification));
+            }
+
+            if (notification == _currentNotification)
+            {
+                // We close the notification. This will trigger the display of the next one.
+                // See OnInfoBarClosed.
+                AssociatedObject.IsOpen = false;
+                return;
+            }
+
+            _stackedNotifications.Remove(notification);
```

Comment:
After adding multiple notifications inside the Sample Page, and after repeatedly clicking the 'Remove last notification' button, I was expecting notifications to be removed one by one.  Instead, it is only the _very last_ notification to be added, that can be removed.  

![image](https://user-images.githubusercontent.com/86266896/146587082-a21be36b-a7f3-4526-bc4e-922f8c82e272.png)


The C# `LinkedList` class serves up a convenient `RemoveLast` method - is that not an option here?  Else, I'd suggest temporarily disabling the 'Remove last notification' button if it's not possible to remove any further notifications.

Reply:
I've removed the button.